Lesson_1/Task1/Task1.4/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+
+namespace Task4
+{
+    class Program
+    {
+        static void Main()
+        {
+
+            string s1 = "hello";
+            string s2 = "world";
+            string s3 = s1 + " " + s2; // результат: строка "hello world"
+            string s4 = String.Concat(s3, "!!!"); // результат: строка "hello world!!!"
```

Comment:
Like in task 3, you should read from the console, also, please reread task 4 more carefully, you should find the index of last matching the symbol with concatenated strings